### ISSUE TITLE
Patched / Not Working?

### ISSUE THREAD

**Issue by SabotageLuke** [RealRainn/justfall.lol-fly-script#1]
As far as I know this script is currently not working as of 1/22/22. Is there a bind I have to press to activate the script? I tried looking but it's all encrypted.

**Comment by RealRainn** [comment]
> As far as I know this script is currently not working as of 1/22/22. Is there a bind I have to press to activate the script? I tried looking but it's all encrypted.

yeah its prob patched, i had this here cause a bunch of kids at school were playing and i just loaded this in console and trolled them, and yes the script looks obfuscated but that's how i got it
